definitions/divorce/json/CaseEventToComplexTypes/CaseEventToComplexTypes-prod.json

```diff
@@ -0,0 +1 @@
+[]
```

Comment:
please remove. This file is sometimes needed. We don't want to created it if it's empty. There are some empty files because they were needed and then they were not removed. In here there is no need to even create it.

Reply:
@yarpo This file along with `AuthrisationComplexTypes` already existed even in master hence thought to leave as is. Should they still be removed? They have existed for some time there now

Reply:
removed